Extensions/DataSources/src/DataSourcesHub/Resources/DataSourcesResources.tsx

```diff
@@ -0,0 +1,8 @@
+export const SUBMIT = 'SUBMIT';
+export const DATA_SOURCES = 'DataSources';
+export const DATA_SOURCE = 'DataSource';
+export const SELECT_DATASOURCE = 'Select a DataSource';
+export const NO_TYPE_MATCH = 'No type match found';
+export const NO_DATASOURCE = 'No DataSources found';
+export const RESULT = 'Result';
+export const STATUS_CODE ='Status Code :'; 
```

Comment:
keep Status Code here 
move logic to show : to UI

Reply:
ok